examples/scripts/trace_example.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the SCICO package. Details of the copyright
+# and user license can be found in the 'LICENSE.txt' file distributed
+# with the package.
+
+r"""
+SCICO Call Tracing
```

Comment:
Should this example appear as a notebook at https://scico--569.org.readthedocs.build/en/569/examples.html? I don't see it there.

Reply:
Intentionally omitted from the notebooks since it's expected to primarily be of interest to developers.